app/cdap/components/ConnectionList/Components/ConnectionTabs/mock/mockTabsDataWithBrowse.tsx

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-import { GCSIcon } from 'components/ConnectionList/icons';
+import { GCSIcon } from 'components/ConnectionList/IconsStore/CGSIcon';
```

Comment:
IconsStore filename has to be iconStore with camelCasing

Reply:
As discussed, renamed it to IconStore